app/src/main/java/pm/gnosis/heimdall/data/repositories/TransactionExecutionRepository.kt

```diff
@@ -23,7 +23,12 @@ interface TransactionExecutionRepository {
     ): Single<ByteArray>
 
     fun loadSafeExecuteState(safeAddress: Solidity.Address, paymentToken: Solidity.Address): Single<SafeExecuteState>
-    fun loadExecuteInformation(safeAddress: Solidity.Address, paymentToken: Solidity.Address, transaction: SafeTransaction): Single<ExecuteInformation>
+    fun loadExecuteInformation(
+        safeAddress: Solidity.Address,
+        paymentToken: Solidity.Address,
+        transaction: SafeTransaction,
+        safeOwner: AccountsRepository.SafeOwner? = null
```

Comment:
Why is safeOwner nullable?

Reply:
I made it nullable (with default null), so that I don't have to adjust every call that already exists